content/article/the-perl-toolchain-summit-2025-needs-you.md

```diff
@@ -0,0 +1,242 @@
+# The Perl Toolchain Summit 2025 needs you
+
+This year in particular,
+[the organizers](https://blogs.perl.org/users/book/2025/02/announcing-the-perl-toolchain-summit-2025.html)
+have had difficulty reaching our fundraising targets for the
+[Perl Toolchain Summit](https://perltoolchainsummit.org/pts2025/).
+
+[In the words of Ricardo Signes](https://rjbs.cloud/blog/2024/05/pts-2024-lisbon/):
+> The Perl Toolchain Summit is one of the most important events in the
+> year for Perl. A lot of key projects have folks get together to get
+> things done.
+
+Everyone who is *invited* to the Summit is a project leader or important
+contributor that is going to give their time and expertise for four
+days, to move the Perl toolchain forward. They give their time
+(sometimes having to take days off work, which is already a loss of
+income or holidays for them).
+
+This is why, since 2011, we've done our best to *at least partially
+refund* their travel and accommodation expenses when needed. Everyone
+who's attending the PTS should really *only* have to give four days of
+their life for it.
+
+If the PTS can't support its participants, then more and more of them
+are going have to either decline our invitation, or spend their own
+money, in addition to their time, to continue supporting the Perl
+Toolchain.
+
+**This is bad for Perl and CPAN.**
+
+Perl is not like Java or Go: it doesn't have a large corporation behind
+it. It's entirely supported by the community and its corporate sponsors.
+In others words, by you.
```

Comment:
While I see some value in specifically naming other programming languages (because readers may have familiarity with how these communities organize their funding, and because leveling with those technologies), I also suggest considering to use a wording without mentioning other tech. For example:

```suggestion
Perl differs from other programming languages which have large corporations
funding their development: it's entirely supported by the community and its sponsors. In other words, by you.
```

ps.: also fixing the possible `s/others/other/` typo.